jetty-util/src/main/java/org/eclipse/jetty/util/AbstractTrie.java

```diff
@@ -29,7 +33,7 @@
  *
  * @param <V> the type of object that the Trie holds
  */
-public abstract class AbstractTrie<V> implements Trie<V>
+abstract class AbstractTrie<V> implements Index.Mutable<V>
```

Comment:
Maybe `AbstractIndex` and the name Trie only applies to real implementations?

Reply:
It doesn't really matter as the class is package-protected, so I'd leave it as it.